internal/controller/rbac/provider/roles/reconciler_test.go

```diff
@@ -395,40 +395,45 @@ func TestClusterRolesDiffer(t *testing.T) {
 
 func TestOrgDiffer(t *testing.T) {
 	cases := map[string]struct {
-		registry string
-		a        string
-		b        string
-		want     bool
+		a    string
+		b    string
+		want bool
 	}{
-		"SameOrg": {
-			registry: "xpkg.example.org",
-			a:        "xpkg.example.org/cool/provider:v1.0.0",
-			b:        "cool/other-provider:v1.0.0",
-			want:     false,
+		"SameOrgWithRegistry": {
+			a:    "xpkg.example.org/cool/provider:v1.0.0",
+			b:    "xpkg.example.org/cool/other-provider:v1.0.0",
+			want: false,
+		},
+		"SameOrgWithNoRegistry": {
```

Comment:
this test case is interesting - is it kind of saying that "it's OK to not specify a registry"?

Reply:
It's not okay in the broader context of Crossplane, but given that the OrgDiffer is its own component with its own "semantic," I thought it made sense to keep it this way. I can be easily convinced to return an error in this case instead.